weis/aeroelasticse/runFAST_pywrapper.py

```diff
@@ -184,9 +296,23 @@ def run_multi(self, cores=None):
         pool.close()
         pool.join()
 
-        return output
+        ss = {}
+        et = {}
+        dl = {}
+        ct = []
+        for _name, _ss, _et, _dl, _ct in output:
+            ss[_name] = _ss
+            et[_name] = _et
+            dl[_name] = _dl
+            ct.append(_ct)
+
+        summary_stats, extreme_table, DELs = la.post_process(ss, et, dl)
+
+        return summary_stats, extreme_table, DELs, ct
 
     def run_mpi(self, mpi_comm_map_down):
+
+        raise NotImplementedError("Function 'run_multi' hasn't been configured to use new post processing routines.")
```

Comment:
I can definitely run in parallel locally with this branch. @JakeNunemaker is this function just no longer used?

Reply:
I think this was from Evan and was an alternate way to run in parallel. It's not used to my knowledge. I added the error because the data structure coming out of eval has changed and this routine hasn't been updated to reflect that. Not sure if it's worth making those changes right now if we have a different parallel processing routine.

Reply:
@gbarter, @JakeNunemaker - just to be clear, does all MPI functionality work with the new methods and without this function call? This is particularly important on eagle, which I _think_ is why it was implemented in the first place.

Reply:
I cleaned this up.  There are three ways to run OpenFAST via AeroelasticSE:

1. Serial with a for loop for multiple runs
2. Multiprocessing library
3. MPI

When running WEIS, MPI is invoked near the beginning of the calling stack to play nicely with OpenMDAO and OpenFAST is executed via number 2.  AeroelasticSE can also be run as a standalone using any of these methods.  This was confusing to sort out because the code was quite messy as Jake pointed out.  I have tried to resolve all of this.

Reply:
We should probably still do some testing on Eagle, but I don't think any prior capability has been lost